tests/system/large/test_streaming.py

```diff
@@ -46,3 +47,29 @@ def test_streaming_to_bigtable():
         assert str(query_job.job_id).startswith(job_id_prefix)
     finally:
         query_job.cancel()
+
+
+def test_streaming_to_pubsub():
+    # launch a continuous query
+    job_id_prefix = "test_streaming_pubsub_"
+    sql = """SELECT
+        island
+        FROM birds.penguins"""
+    query_job = bigframes.streaming.to_pubsub(
+        sql,
+        topic="penguins",
+        service_account="streaming-testing@bigframes-load-testing.iam.gserviceaccount.com",
+        job_id=None,
+        job_id_prefix=job_id_prefix,
+    )
+
+    try:
+        # wait 100 seconds in order to ensure the query doesn't stop
+        # (i.e. it is continuous)
+        time.sleep(100)
+        assert query_job.error_result is None
+        assert query_job.errors is None
+        assert query_job.running()
+        assert str(query_job.job_id).startswith(job_id_prefix)
```

Comment:
Thoughts on checking that we wrote at least one message to the pub/sub topic?

Reply:
By default, we wouldn't write any message to the pubsub topic, if I understand correctly. We would have to write to the BigQuery table in order for subscribers to receive anything. We would have to create a subscription externally and listen in this test using the client library. That would protect against any situation where the query keeps running but fails silently at actually transmitting data, but I don't think there is any situation like that, especially not one that would be a BigFrames bug rather than a continuous queries bug. Does that make sense?

Reply:
I'm still not sure we need it, though. We're really just launching a continuous query and as long as the query was created without errors and is indeed continuous, to me what data gets exported is the realm of continuous queries behavior and the user's SQL. (And possibly destination configuration which we are also hands-off about)